homeassistant/components/tplink/strings.json

```diff
@@ -68,21 +68,6 @@
       },
       "today_consumption": {
         "name": "Today's consumption"
-      },
-      "current_consumption_child": {
-        "name": "{device_name} current consumption"
-      },
-      "total_consumption_child": {
-        "name": "{device_name} total consumption"
-      },
-      "today_consumption_child": {
-        "name": "{device_name} today's consumption"
-      },
-      "current_child": {
-        "name": "{device_name} current"
-      },
-      "voltage_child": {
-        "name": "{device_name} voltage"
```

Comment:
I suppose these are now unnecessary as we expose the children as their own devices? If so, that's great :+1: